manifests/base/gateway-controller/gateway-controller-deployment.yaml

```diff
@@ -16,8 +15,7 @@ spec:
       serviceAccountName: argo-events-sa
       containers:
         - name: gateway-controller
-          image: argoproj/gateway-controller:v0.14.0
-          imagePullPolicy: Always
+          image: argoproj/gateway-controller:latest
```

Comment:
I am not very comfortable with the latest tag images. Can we pin to a version?

Reply:
I've made the image tags [customizable](https://kubectl.docs.kubernetes.io/pages/app_customization/customizing_pod_templates.html#customizing-images), so it will be easier to maintain - both controller's image tags are enforced in `base/kustomization.yaml` - together, at the same place. Default is left to latest in the `base/sensor-controller/sensor-controller-deployment.yaml` and `base/gateway-controller/gateway-controller-deployment.yaml`. Would that be a suitable solution?

Reply:
that works